server/bin/unittests

```diff
@@ -59,6 +59,12 @@ function _run_indexing {
     _run pbench-index --tool-data
 }
 
+function _run_pytest {
+    tname=${1}
+    export PYTHONPATH=${_testopt}/lib:$PYTHONPATH
+    pytest ${_testopt}/pytests/${tname}.py
```

Comment:
1) Wrap the run with `echo ...` similar to what `_run` does.
2) Redirect the output of the test to `${_testout}` similar to what `_run` does.
3) Run pytest with -q to quiet down its output. The normal output prints out python versions which we don't want in the output file: they are going to cause spurious failures if somebody is using a slightly different version of python or pytest.